timeseries/src/autogluon/timeseries/models/presets.py

```diff
@@ -30,6 +29,9 @@
 from .abstract import AbstractTimeSeriesModel
 from .multi_window.multi_window_model import MultiWindowBacktestingModel
 
+# TODO: Enable ARIMAModel after upgrading to StatsForecast >=1.5.0 - currently ARIMA model is broken
```

Comment:
Because of the bug described above (in-place modification of `DUMMY_MODEL_HPARAMS`), we missed the fact that `ARIMA` model doesn't work in StatsForecast v1.4.0 — the model failure was caught by the fallback model. We will enable the model after we resolve #3458 and upgrade StatsForecast to >= v1.5.0.